packages/components/pie-chip/src/index.ts

```diff
@@ -61,6 +64,7 @@ export class PieChip extends LitElement implements ChipProps {
     private renderCloseButton (): TemplateResult {
         return html`
                     <button
+                        @click="${this.disabled ? null : () => dispatchCustomEvent(this, ON_CHIP_CLOSE_EVENT)}"
```

Comment:
If you were to split it like this, does the click handler still get called when the button is disabled?
```suggestion
                        @click="{() => dispatchCustomEvent(this, ON_CHIP_CLOSE_EVENT)}"
                        ?disabled={this.disabled}
```

Reply:
Great suggestion! I updated the code 🙇‍♀️